### PR TITLE
[issue_tracker] update test plan

### PR DESCRIPTION
**Pull request by ridz1208**
## Brief summary of changes
testplan

### CODE REVIEW

File: modules/issue_tracker/test/issue_tracker_test_plan.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # Issue Tracker Test Plan
 
 ## Issue Tracker Filter Form [Automation Testing]
-1. User can access the page only if they have `issue_tracker_reporter` or `issue_tracker_developer` permission
+1. User can access the page only if they have `Issue Tracker: Close/Edit/Re-assign/Comment on All Issues` or ` Issue Tracker: Create/Edit Own Issues and Comment on All Issues` permission
```
Comment (ridz1208) [inline modules/issue_tracker/test/issue_tracker_test_plan.md:4]: @charliehenrib these will need to still include the backend names. you can add them in parenthesis if you want.

Comment (ridz1208) [reply modules/issue_tracker/test/issue_tracker_test_plan.md:4]: choose a convention like the one below and stick to it maybe

```
_Front end label name_ (`backend_permission`)
```

File: modules/issue_tracker/test/issue_tracker_test_plan.md [hunk 2]
```diff
@@ -27,9 +27,9 @@
 
 ## Issue Tracker Edit Existing Issue [Manual Testing]
 1. User can access the page if they fulfill all the following conditions:
-	* they have `issue_tracker_developer` or `issue_tracker_reporter` permission 
+	* they have `Issue Tracker: Close/Edit/Re-assign/Comment on All Issues` or ` Issue Tracker: Create/Edit Own Issues and Comment on All Issues` permission 
 	* they have `access_all_profiles` or are a member of the site of the issue or the site has no issue.
-2. Users who have `issue_tracker_reporter` permission can edit all fields if it is their issue, but are blocked except for commenting and watching options for all other issues. Users with `issue_tracker_developer` permission can make all changes on all issues that they can view. 
+2. Users who have ` Issue Tracker: Create/Edit Own Issues and Comment on All Issues` permission can edit all fields if it is their issue, but are blocked except for commenting and watching options for all other issues. Users with `Issue Tracker: Close/Edit/Re-assign/Comment on All Issues` permission can make all changes on all issues that they can view.
```
Comment (cmadjar) [inline modules/issue_tracker/test/issue_tracker_test_plan.md:32]:
```suggestion
2. Users who have `Issue Tracker: Create/Edit Own Issues and Comment on All Issues` permission can edit all fields if it is their issue, but are blocked except for commenting and watching options for all other issues. Users with `Issue Tracker: Close/Edit/Re-assign/Comment on All Issues` permission can make all changes on all issues that they can view. 
```

Comment (cmadjar) [reply modules/issue_tracker/test/issue_tracker_test_plan.md:32]: don't judge me. My eyes caught that and I had to ask for a change. ;)

File: modules/issue_tracker/test/issue_tracker_test_plan.md [hunk 3]
```diff
@@ -43,8 +43,8 @@
 
 ## Permissions [Automation Testing]
 1. Remove `access_all_profiles` permission.
-2. Remove `issue_tracker_reporter` permission
-3. Remove `issue_tracker_developer` permission
+2. Remove ` Issue Tracker: Create/Edit Own Issues and Comment on All Issues` permission
```
Comment (cmadjar) [inline modules/issue_tracker/test/issue_tracker_test_plan.md:46]:
```suggestion
2. Remove `Issue Tracker: Create/Edit Own Issues and Comment on All Issues` permission
```

**Review by cmadjar — CHANGES_REQUESTED** [review]
Two very difficult changes asked ;)

File: modules/issue_tracker/test/issue_tracker_test_plan.md (hunk 2 above)
Comment (ridz1208) [reply modules/issue_tracker/test/issue_tracker_test_plan.md]: damn!!

**Comment by ridz1208** [comment]
@cmadjar done !

**Review by cmadjar — APPROVED** [review]
👍

**Pull request merged**